# mobileOG-db | Release name: beatrix-1.6

## ![alt text](https://i.imgur.com/XMuXfit.jpeg)

## Description of mobileOG-db
mobileOG-db (for mobile orthologous groups) is a manually curated database of protein families mediating the integration/excision, replication/recombination/repair,
stability/defense, or transfer of bacterial mobile genetic elements and phages as well as the associated transcriptional regulators of these processes. mobileOG-db was created
in a process involving more than 15,000 unique manual curation events, with 1,558 references and 2,444 unique manual functional annotations at present.

## Try mobileOG-pl, a pipeline to detect and categorize bacterial mobile genetic elements from contigs or long reads. 
https://github.com/clb21565/mobileOG-db/blob/main/mobileOG-pl/UsageGuidance.md

## Downloads 
The current version of the database can be browsed accessed through its website: 
https://mobileogdb.flsi.cloud.vt.edu/

## Contact Us

connorleebrown@gmail.com, mobileogdbsupport@vt.edu

## Citing Us

mobileOG-db: a manually curated database of protein families mediating the life cycle of bacterial mobile genetic elements
Connor L. Brown, James Mullet, Fadi Hindi, James E. Stoll, Suraj Gupta, Minyoung Choi, Ishi Keenum, Peter Vikesland, Amy Pruden, Liqing Zhang
bioRxiv 2021.08.27.457951; doi: https://doi.org/10.1101/2021.08.27.457951

## Acknowledgements 
To create mobileOG-db, we analyzed 10,776,212 proteins derived from 8 different databases of complete MGE sequences. The references to these databases are below. 

**1. ICEBerg (ICEs, AICEs, CIMEs, IMEs):**

Meng Liu, Xiaobin Li, Yingzhou Xie, Dexi Bi, Jingyong Sun, Jun Li, Cui Tai, Zixin Deng, Hong-Yu Ou, ICEberg 2.0: an updated database of bacterial integrative and conjugative elements, Nucleic Acids Research, Volume 47, Issue D1, 08 January 2019, Pages D660â€“D665, https://doi.org/10.1093/nar/gky1123

**2. ACLAME (various):**

Leplae R, Lima-Mendez G, Toussaint A. ACLAME: a CLAssification of Mobile genetic Elements, update 2010. Nucleic Acids Res. 2010 Jan;38(Database issue):D57-61. doi: 10.1093/nar/gkp938. Epub 2009 Nov 23. PMID: 19933762; PMCID: PMC2808911.

**3. GutPhage Database (Bacteriophages derived from human gut metagenomes):**

Camarillo-Guerrero LF, Almeida A, Rangel-Pineros G, Finn RD, Lawley TD. Massive expansion of human gut bacteriophage diversity. Cell. 2021 Feb 18;184(4):1098-1109.e9. doi: 10.1016/j.cell.2021.01.029. PMID: 33606979; PMCID: PMC7895897.

**4. Prokaryotic viral orthologous groups (pVOG) (a collection of prokaryotic virus protein HMMs):**


Grazziotin AL, Koonin EV, Kristensen DM. Prokaryotic Virus Orthologous Groups (pVOGs): a resource for comparative genomics and protein family annotation. Nucleic Acids Res. 2017;45(D1):D491-D498. doi:10.1093/nar/gkw975

**5. COMPASS (plasmids):**


Douarre PE, Mallet L, Radomski N, Felten A, Mistou MY. Analysis of COMPASS, a New Comprehensive Plasmid Database Revealed Prevalence of Multireplicon and Extensive Diversity of IncF Plasmids. Front Microbiol. 2020;11:483. Published 2020 Mar 24. doi:10.3389/fmicb.2020.00483

**6.** NCBI Plasmid RefSeq (plasmids): ****

O'Leary NA, Wright MW, Brister JR, et al. Reference sequence (RefSeq) database at NCBI: current status, taxonomic expansion, and functional annotation. Nucleic Acids Res. 2016;44(D1):D733-D745. doi:10.1093/nar/gkv1189

**7. **immedb (integrative elements):****

Jiang,X., Hall,A.B., Xavier,R.J. and Alm,E.J. (2019) Comprehensive analysis of chromosomal
mobile genetic elements in the gut microbiome reveals phylum-level niche-adaptive
gene pools. PLoS One, 14, e0223680.

**8. ISfinder (insertion sequences):**

Siguier P, Perochon J, Lestrade L, Mahillon J, Chandler M. ISfinder: the reference centre for bacterial insertion sequences. Nucleic Acids Res. 2006 Jan 1;34(Database issue):D32-6. doi: 10.1093/nar/gkj014. PMID: 16381877; PMCID: PMC1347377.
